### PR TITLE
feat: deep link myinfo settings message in builder

### PR DESCRIPTION
**Pull request by foochifa**
## Problem

With deeplinking, we can just link admins to settings on the myinfo message, similar to payments!

## Solution

**Breaking Changes** 

- [ ] No - this PR is backwards compatible  

## Before & After Screenshots

**AFTER**:

image

## Tests

- [ ] Create an email mode form
- [ ] Go to `create` tab, and select `myinfo` tab, you should see the message as the screenshot above
- [ ] Settings should be clickable
- [ ] Click settings, enable myinfo
- [ ] Add myinfo field
- [ ] Open form
- [ ] Fill and submit form

test payment settings redirect still works
- [ ] Create a storage mode form
- [ ] Go to `create` tab and select `payment` tab with stripe unconnected
- [ ] You should be able to select 'Settings' in the info message
- [ ] It should redirect you to `settings/payments`

### CODE REVIEW

**Comment by foochifa** [comment]
@staceytan1998 followed the design of payments message, is this okay?

File: frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/MyInfoPanel.tsx [hunk 1]
```diff
@@ -139,7 +140,13 @@ const MyInfoText = ({
 
   if (isMyInfoDisabled) {
     return (
-      <Text>Enable MyInfo in the Settings tab to access these fields.</Text>
+      <Text>
+        Enable MyInfo in the{' '}
+        <Link as={ReactLink} to={`settings/singpass`}>
```
Comment (KenLSM) [inline frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/MyInfoPanel.tsx]: Might make sense make this as a constant here
```
// frontend/src/constants/routes.ts
export const ADMINFORM_SETTINGS_SINGPASS_SUBROUTE= `${ADMINFORM_SETTINGS_SUBROUTE}/singpass`
```

**Review by KenLSM — APPROVED** [review]
LGTM!

File: frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/MyInfoPanel.tsx (hunk 1 above)
Comment (foochifa) [reply frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/MyInfoPanel.tsx]: [39abc1d](https://github.com/opengovsg/FormSG/pull/6503/commits/39abc1d6845b80672b80e52f4c4cb33e49697490)

Thanks for the suggestion @KenLSM! Have also made an update to payment's settings link

**Pull request merged**